doc/input-output-reference/src/input-for-output.tex

```diff
@@ -429,7 +429,7 @@ \subsubsection{Inputs}\label{inputs-8-019}
 Output:Variable, * , Zone Air System Sensible Heating Energy, hourly;
 Output:Variable, * , Zone Air System Sensible Cooling Energy, hourly;
 Output:Variable, * , Zone Air Temperature, hourly;
-Output:Variable, * , Zone Air Temperature;  ! same as previous line
+Output:Variable, * , Zone Air Temperature;  !- same as previous line
```

Comment:
Maybe it is ok to leave everything on a single line for Output:Variable since that is kind of an "endorsed" format.